docs/2.9.0/docs/canton/usermanual/pruning.rst

```diff
@@ -40,9 +44,81 @@ Refer to the cron specification to customize the pruning schedule. Here are a fe
    :dedent:
 
 For the maximum duration to specify a reliable pruning window "end time", the leading fields of the cron expression
-should should not be wildcards (`*`) as illustrated in the examples above. If the hour field is fixed, so should the
+should not be wildcards (`*`) as illustrated in the examples above. If the hour field is fixed, so should the
 fields for minute and second.
 
+Schedule format
+---------------
+
+Cron expressions are formatted as 7 whitespace-separated fields:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Field
+     - Type and valid values\*
+   * - seconds
+     - number from ``0`` to ``59``
+   * - minutes
+     - number from ``0`` to ``59``
+   * - hours
+     - number from ``0`` to ``23``
+   * - day of the month
+     - number from ``1`` to ``31``
+   * - month
+     - number from ``0`` to ``11`` (``0`` = January) 
+       |br|
+       **or** 
+       |br|
+       the first three letters of the month name (``JAN``, ``FEB``, ``MAR``, ...)
+   * - day of the week
+     - number from ``1`` to ``7`` (``1`` = Sunday) 
+       |br|
+       **or** 
+       |br|
+       the first three letters of the day name (``SUN``, ``MON``, ``TUE``, ...)
+   * - year (optional)
+     - number from ``1900`` to ``2099``
+
+*\*Ranges specified with "to" are inclusive of both endpoints.*
```

Comment:
This sentence threw me off - it took me a while to figure out that it refers to the ranges specified in the table above, and not to ranges the user would define.

How bad would it be layout-wise to just say "included" in all relevant entries in the table directly? (And then remove this sentence.) Alternatively,

```suggestion
*\*Ranges specified with "to" in the table above are inclusive of both endpoints.*
```

Reply:
Hmm. Were you looking at just the .rst or also the rendered view? There's an asterisk that connects the column heading to the note (you can also see it here https://docs.daml.com/3.0.0/canton/usermanual/pruning.html#schedule-format)

Maybe I'll go with "In the table, ranges specified..." 

<img width="709" alt="table" src="https://github.com/digital-asset/docs.daml.com/assets/139921049/8bbe16c9-904a-4007-9b68-b70eae7ec809">

Reply:
I can also look at putting the note right there under the column heading, though I think I tried that and it was messy. But I'll check.

Reply:
Ah, sorry, no, I was just reading through the rst. It is indeed a lot clearer with the layout.